examples/data_pipeline/pipelines.py

```diff
@@ -51,37 +51,40 @@ def build_batch_torchtext_vocab(vocab_file):
     tokenizer = get_tokenizer("basic_english")
     from torchtext.vocab import build_vocab_from_iterator
     from transforms import TextClassificationPipeline
+    from torchtext.experimental.functional import totensor, vocab_func, sequential_transforms
 
     def token_iterator(vocab_file):
         f = open(vocab_file, 'r')
         for token in f:
             yield token
     vocab = build_vocab_from_iterator(token_iterator(vocab_file))
-    text_pipeline = TextDataPipeline(tokenizer, partial(map, vocab))
-    label_pipeline = int
+    text_pipeline = sequential_transforms(tokenizer, vocab_func(vocab), totensor(dtype=torch.long))
+    label_pipeline = totensor(dtype=torch.long)
     return TextClassificationPipeline(label_pipeline, text_pipeline), None
 
 
-def build_huggingface_vocab_pipeline(hf_vocab_file):
+def build_text_vocab_pipeline(hf_vocab_file):
     tokenizer = BasicEnglishNormalize()
     f = open(hf_vocab_file, 'r')
     vocab = vocab_from_file_object(f)
 
     # Insert token in vocab to match a pretrained vocab
-    pipeline = TextDataPipeline(tokenizer, VocabTransform(vocab))
+    pipeline = TextSequentialTransforms(tokenizer, VocabTransform(vocab), ToLongTensor())
     jit_pipeline = torch.jit.script(pipeline)
     print('jit Hugging Face pipeline success!')
     return pipeline, jit_pipeline
 
 
 def build_pytext_vocab_pipeline(vocab_file):
+    from pytext.torchscript.vocab import ScriptVocabulary
```

Comment:
Is there a reason you do the import in the functions instead of at the top of the file?

Reply:
If we don't run `build_pytext_vocab_pipeline` func, we don't need to add pytext dependency.